README.rst

```diff
@@ -208,6 +208,11 @@ using the ``WATCHMAN_ERROR_CODE`` setting::
 
     WATCHMAN_ERROR_CODE = 500
 
+Logging
+*******
+
+watchman includes log messages using a logger called ``watchman.decorators``.
```

Comment:
Are we able to set this name? I think using `logging.getLogger(__name__)` is normal in an app where you could be logging from many different places and you want to capture that automatically. In this case having a logger called `watchman` (if possible) would probably be nicer.

Reply:
Good call, I have updated to use that. I was thinking about in the future when there might be more logging happening and users might want to configure different loggers in different ways, but it's easy enough to update to have multiple loggers when/if it's ever actually needed.